frontend/tasks.py

```diff
@@ -3272,15 +3274,41 @@ def get_rel_energy(s):
 
 
 @app.task(base=AbortableTask)
-def dispatcher(order_id):
-    order = CalculationOrder.objects.get(pk=order_id)
-    ensemble = order.ensemble
+def dispatcher_flowchart():
```

Comment:
For debugging only, I suppose? To remove if it is no longer necessary

Reply:
Yes, it was for debugging only. Removed.